crates/re_types/definitions/rerun/archetypes/series_point.fbs

```diff
@@ -26,4 +26,9 @@ table SeriesPoint (
     ///
     /// Used in the legend.
     name: rerun.components.Name ("attr.rerun.component_optional", nullable, order: 3000);
+
+    /// Size of the markers.
+    ///
+    /// Can set a single one for all markers or a batch of sizes.
```

Comment:
Isn't it declared as a mono right below? How could it be a batch?

Reply:
urgh, thanks. Was too hasty!